week5part3.py

```diff
@@ -0,0 +1,26 @@
+def all_perfect_numbers():
+    """
+    Generator function that ran to infinity
```

Comment:
I think that "run infinite amount of times" would be more suitable.

Reply:
run infinite amount of times